modules/avatar/react/lib/AvatarButton.tsx

```diff
@@ -47,6 +47,21 @@ export default class AvatarButton extends React.Component<AvatarButtonProps> {
     altText: 'Avatar',
   };
 
+  constructor(props: AvatarButtonProps) {
+    super(props);
+    this.state = {
+      hasImageError: false,
+    };
+  }
```

Comment:
Are you okay with the following instead?

```suggestion
  state = {
    hasImageError: false,
  }
```

It removes the `readonly` from `state`, but it is much cleaner.